### PR TITLE
fix: added support for static tools in pin-3.26

### PR DESCRIPTION
**Pull request by saipavanmeruga**
Made changes to support the compilation of Static pintools using pin-3.26 on Linux Platforms.

### CODE REVIEW

**Review by saipavanmeruga — COMMENTED** [review]
These changes will add support for building Static Pintools in pin-3.26 on linux platforms.

File: config/pin_static.mpb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 project : pin {
+requires += static
```
Comment (jwillemsen) [inline config/pin_static.mpb:2]: Fix indent

File: config/pintool_static.mpb [hunk 2]
```diff
@@ -26,7 +25,7 @@ project : pin_static {
     else
       # TODO Add support for linking against gcc libraries included with Pin
       CPPFLAGS += -DTARGET_LINUX
-      LIBS     += -lpindwarf
+      LIBS     += -static
```
Comment (jwillemsen) [inline config/pintool_static.mpb:28]: Libs doesn’t look the correct place for -static

File: config/pin_base.mpb [hunk 3]
```diff
@@ -42,10 +42,11 @@ project {
 
   specific (gnuace, make) {
     compile_flags += -O3 -funwind-tables -fno-stack-protector -fasynchronous-unwind-tables -fomit-frame-pointer -fno-strict-aliasing -fno-rtti -fPIC -nostdlib -fpermissive -Wno-error=all -fno-exceptions
-    compile_flags -= -Wunused-parameter
+    compile_flags -= -Wunused-parameter -nostdlib
 
     libs += xed
-    libs += c-dynamic m-dynamic stlport-dynamic unwind-dynamic 
+    libs += c-dynamic m-dynamic c++ c++abi unwind-dynamic
+    ldlibs =
```
Comment (jwillemsen) [inline config/pin_base.mpb:49]: Why set it empty?

**Review by saipavanmeruga — COMMENTED** [review]
Resolved the comments provided by maintainers.

**Pull request merged**